### ISSUE TITLE
Quick Prediction does segmentation but image completion fails

### ISSUE THREAD

**Issue by JayKarhade** [sujaykhandekar/Automated-objects-removal-inpainter#4]
I was running the quick prediction on google colab after downloading the pre-trained models, however, i just get weird green results at the segmented parts of the image. 
![32ec0bb3-8c56-4128-9b9a-15a5b0074fee](https://user-images.githubusercontent.com/48925233/123999047-b7438a00-d9ef-11eb-9409-b9bc1c9fd498.jpeg)
![237d7823-c332-4c01-98b7-8ccfa92cdf03](https://user-images.githubusercontent.com/48925233/123999168-d6421c00-d9ef-11eb-956e-c5073cf1d684.jpeg)

I am using pytorch 1.8.1 and all other dependencies are same as the ones specified.

Any reason why this could be happening?

**Comment by lsh950919** [comment]
Hi @JayKarhade,

Did you happen to find the solution to this? I'm new on this project and the same thing happened to me when I ran the test.py code according to the README of this repo.

It would be great if you can tell me how you fixed the issue.

Thanks!

**Comment by JayKarhade** [comment]
Hi, @lsh950919, yeah the problem for me was using the bash script. you'll have to manually transfer the contents of the folders after downloading. 

Alternatively, you can download the models manually in the folders. That did the trick for me.

**Comment by Vnk0410** [comment]
While running the code,am also getting the green color masked image. Please guide me how to achieve the result.

**Comment by vineeth12365** [comment]
> While running the code,am also getting the green color masked image. Please guide me how to achieve the result.

Bro did U resolved your issue?

**Comment by vineeth12365** [comment]
> I was running the quick prediction on google colab after downloading the pre-trained models, however, i just get weird green results at the segmented parts of the image. ![32ec0bb3-8c56-4128-9b9a-15a5b0074fee](https://user-images.githubusercontent.com/48925233/123999047-b7438a00-d9ef-11eb-9409-b9bc1c9fd498.jpeg) ![237d7823-c332-4c01-98b7-8ccfa92cdf03](https://user-images.githubusercontent.com/48925233/123999168-d6421c00-d9ef-11eb-956e-c5073cf1d684.jpeg)
> 
> I am using pytorch 1.8.1 and all other dependencies are same as the ones specified.
> 
> Any reason why this could be happening?

Bro did U resolved your issue?
